src/components/Table/Table.stories.mdx

```diff
@@ -36,10 +39,12 @@ I'm a table!
         { id: 3, color: 'green', flavor: 'strawberry' },
       ];
       return (
-        <Table
-          columns={columnConfig}
-          tableData={tableData}
-        />
+        <Table>
+          <TableHead columns={columnConfig} />
+          <TableBody>
+            {tableData.map(row => <TableRow data={row} key={row.id} />)}
```

Comment:
Console still has missing key errors. I think key needs to be received as a prop by TableRow, and this key applied to the `<td>` elements.

Reply:
Hmm the <tr> already have a key though. Checking.